source/styleEditor.js

```diff
@@ -1,3 +1,7 @@
 $(document).ready(function(){
-  // your code goes here.
+  $('#style_editor').on('submit', function(e){
+    e.preventDefault();
+    var $data = $(this).serializeArray();
```

Comment:
While .serializeArray() does work, I'd say readability is at a loss here. I'd suggest using jQuery's .val() method on an input selector just to be more explicit about the values you're pulling from the DOM